docs/advanced/storing-login-details.md

```diff
@@ -105,8 +137,8 @@ rpcpass=<password>
 
 ## **<i class="fa fa-tasks"></i> Additional Configuration Options**
 
-All command line options can be put in the config file.
+All command line startup options can be put in the config file.
```

Comment:
I don't see how it adds clarity.  It just seems confusing to me since it isn't really adding any new information.